src/common/compat.c

```diff
@@ -70,6 +67,11 @@
 #include <sys/capability.h>
 #endif
 
+/* Now deprecated in Linux GLIBC */
+#if HAVE_SYS_SYSCTL_H && !defined(_WIN32) && !defined(__linux__)
```

Comment:
This should check "defined(HAVE_SYS_SYSCTL_H)", not "HAVE_SYS_SYSCTL_H" -- it causes appveyor failures otherwise.